sdks/agent-sdk/src/core/Agent.ts

```diff
@@ -106,6 +108,11 @@ export type AgentErrorMiddleware<ContentTypes = unknown> = (
   next: (err?: unknown) => Promise<void> | void,
 ) => Promise<void> | void;
 
+export type AgentStreamingOptions<ContentTypes> = Omit<
+  StreamOptions<Message, DecodedMessage<ContentTypes>>,
```

Comment:
the generic values of `StreamOptions` are not necessary here as they are only used in the `onValue` callback.